SU-SWS/saa_victoria_subtheme
# Adapt 284  icon card styles with colored tabs

# READY FOR REVIEW

# Summary
- Made all cards use the icon card class (temporarily until we get interface settings)
- Add colored tag styles for cards, rotating through color styles
- setting the right image style for the icon images 
- setting image size max

# Review By (Date)
- this week, ideally by June 5th please

# Urgency
- Soon

# Steps to Test

1. Pull down branch,
1. run `npm run build` in the theme folder
2.  create a basic page with 2 rows of 3 cards, 2 rows of 2 cards, and 3 rows of 1 card, each with small icons (1:1 ratio) that you can download from figma design: https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=192%3A628
3. see card styles matching figma designs ^
Note that it is just padding tweaks and colored corners.
4. Remove an icon from a card so it doesn't have an image, and the padding on top should still look nice with spacing above the card header. (4 modular spacing above card title in that case)
Note: the red link styles on CTA links and the fonts have not yet been applied, those will be applied generally in another task.

# Affected Projects or Products
- Just SAA subtheme (not used anywhere yet)

# Associated Issues and/or People
- ADAPT-284

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
